modules/3d/include/opencv2/3d/depth.hpp

```diff
@@ -0,0 +1,1264 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+#ifndef __OPENCV_RGBD_DEPTH_HPP__
+#define __OPENCV_RGBD_DEPTH_HPP__
```

Comment:
No underscores + upper case is allowed in public headers. Issue: #7288

Reply:
Fixed